internal/receiver/discoveryreceiver/README.md

```diff
@@ -0,0 +1,334 @@
+# Discovery Receiver
+
+| Status                   |                  |
+| ------------------------ | ---------------- |
+| Stability                | [in-development] |
+| Supported pipeline types | logs             |
+| Distributions            | [Splunk]         |
+
+The Discovery receiver is a logs pipeline compatible receiver that provides the ability to test the functional
```

Comment:
How about: "The Discovery receiver is a receiver compatible with logs pipeline which allows you to test the functional status..."